Weather App/homepage.html

```diff
@@ -10,10 +10,10 @@
 </head>
 <body>
 <div class="container">
-    <div class="search">
-        <input type="text" placeholder="Enter city name" spellcheck="false">
-        <button><img src ="search.png"></button>
-    </div>
+    <form class="search" id="searchForm">
+        <input type="text" placeholder="Enter city name" spellcheck="false" name="cityName">
+        <button type="submit"><img src ="search.png"></button>
+    </form>

```

Comment:
@Parakram07 We need to use form tags to ensure the user submits the city name by pressing the enter button.